hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1385,6 +1387,46 @@ def minimize_nodes(self, nodes):
         else:
             raise NotImplementedError
 
+    @defines_shrink_pass()
+    def zero_examples(self, chooser):
```

Comment:
```suggestion
    def try_trivial_examples(self, chooser):
```

I think `zero` is confusing terminology now that the choice sequence is typed.